src/components/ModalWindow/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import MuiDialogTitle from '@material-ui/core/DialogTitle';
+import MuiDialogContent from '@material-ui/core/DialogContent';
+import MuiDialogActions from '@material-ui/core/DialogActions';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import Typography from '@material-ui/core/Typography';
+
+const styles = (theme) => ({
+  root: {
+    display: 'block',
+    margin: 0,
+    padding: theme.spacing(2),
+  },
+  closeButton: {
+    position: 'absolute',
+    right: theme.spacing(1),
+    top: theme.spacing(1),
+    color: theme.palette.grey[500],
+  },
+});
+
+const DialogTitle = withStyles(styles)((props) => {
+  const { children, classes, onClose, ...other } = props;
+  return (
+    <MuiDialogTitle disableTypography className={classes.root} {...other}>
+      <Typography variant="h6">{children}</Typography>
+      {onClose ? (
+        <IconButton
+          aria-label="close"
+          className={classes.closeButton}
+          onClick={onClose}
+        >
+          <CloseIcon />
+        </IconButton>
+      ) : null}
+    </MuiDialogTitle>
+  );
+});
+
+const DialogContent = withStyles((theme) => ({
+  root: {
+    padding: theme.spacing(2),
+  },
+}))(MuiDialogContent);
+
+const DialogActions = withStyles((theme) => ({
+  root: {
+    margin: 0,
+    padding: theme.spacing(1),
+  },
+}))(MuiDialogActions);
+
+export default function ModalWindow(modalParams) {
+  const [open, setOpen] = React.useState(false);
+
+  useEffect(() => {
+    if (modalParams.modalParams.isActive) {
+      handleClickOpen();
+    }
+  }, [modalParams.modalParams]);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <div>
+      <Dialog
+        fullWidth={true}
+        onClose={handleClose}
```

Comment:
Why do you need to use an additional function for it?

```js
onClose={handleClose}
```

instead of

```js
onClose={() => { setOpen(false); }}
```

Reply:
https://material-ui.com/ru/components/dialogs/
![Screenshot_1](https://user-images.githubusercontent.com/17178553/90287426-88c4cb00-de80-11ea-85c7-031d067e9092.png)

This solution offers MaterialUI

Reply:
Yeah, but do you need it?